sql/driver/conn.go

```diff
@@ -111,6 +108,12 @@ func (c *conn) send(args Request) (*rows, error) {
 	if index < 0 {
 		return r, nil
 	}
+	// Check for any application errors.
+	for _, result := range resp.Results {
+		if result.Error != nil {
+			return nil, result.Error
```

Comment:
I'll add a comment. I don't this is worth it at this stage. It's not even clear if anyone would want to send multiple statements in a single request. Perhaps someone might have a batch workload that could bunch statements from a stream in the future.